ssm/parameters_test.go

```diff
@@ -110,15 +110,37 @@ func (m *mockSSMClient) DescribeParametersWithContext(ctx context.Context, input
 		return nil, m.err
 	}
 
-	return &ssm.DescribeParametersOutput{
-		Parameters: []*ssm.ParameterMetadata{
-			&ssm.ParameterMetadata{
-				Name:             testParam1.Param.Name,
-				KeyId:            testParam1.Param.ARN,
-				LastModifiedDate: testParam1.Param.LastModifiedDate,
-			},
-		},
-	}, nil
+	for _, p := range []testParam{testParam1, testParam2, testParam3} {
+		if org+"/"+prefix+"/"+aws.StringValue(p.Param.Name) == aws.StringValue(input.ParameterFilters[0].Values[0]) {
+			return &ssm.DescribeParametersOutput{
+				Parameters: []*ssm.ParameterMetadata{
+					&ssm.ParameterMetadata{
+						Name:             p.Param.Name,
+						KeyId:            p.Param.ARN,
```

Comment:
I assume this `KeyId` doesn't really matter, you're just using the parameter ARN as a dummy value

Reply:
oh yeah i didnt fix that in the test.  in real life I thought we might want the keyid down the line, so I didn't take it out.